terraform/backend_setup/main.tf

```diff
@@ -1,6 +1,5 @@
 resource "aws_s3_bucket" "terraform" {
-  // TODO Add a prefix to the bucket name (e.g. com.example.myproject.terraform). This should also be in terraform.tf.
-  bucket = "<prefix>.woeip.terraform"
+  bucket = "openoakland.woeip.terraform"
```

Comment:
S3 buckets must be globally unique. This bucket will keep track of the Terraform state files so that we know what infrastructure has been deployed.